### DISCUSSION TITLE
Is it possible to run SCENIC+ with only scATAC data (without scRNA)?

### DISCUSSION THREAD

**Discussion by Ziyan-Lin** [aertslab/scenicplus#164] [Q&A]
Hi SCENIC+ team, 

Thanks for the amazing tool! I have couple samples only have scATAC-seq data but not scRNA-seq data. I am wondering would it still be possible to run SCENIC+ on those samples? Can I for example use the gene activity matrix from Signac as the  gene expression matrix of scRNA side? 
I would be really appreciated if anyone have any suggestions!

Many thanks,
Ziyan

**Comment by WeiFeng0726** [comment]
I have same problem as you, i don't know if it makes sense, but it's works

**Comment by SeppeDeWinter** [comment]
Hi @Ziyan-Lin and @fatcat-del 

Yes, this becomes a bit more complicated indeed.
If you are able to link the cells for which you only have scATAC-seq to cell types/states in the scRNA-seq you are able to use them.
See this tutorial for example: https://scenicplus.readthedocs.io/en/latest/mix_melanoma_cell_lines.html.

But this is not super trivial.

I hope this helps,

All the best!

Seppe

**Comment by AnasAnsari123** [comment]
Hi, is this tutorial is still avilable? I am unable to open this tutorial. It would be great if I get the tutorial file, because i am also having the scATAC data.

**Reply by WeiFeng0726** [reply]
it‘s not avilable. you can follow tutorial, and scRNA data could be replaced by RNA activity matrix which inferred by scATAC

**Reply by SeppeDeWinter** [reply]
Hi both,

You can still find the tutorial here: https://github.com/aertslab/scenicplus/blob/old/notebooks/mix_melanoma_cell_lines.ipynb

Best,
Seppe

**Comment by Melody-cell** [comment]
Hi, @SeppeDeWinter 
I carefully read the tutorial, but how can I get the input data?
![image](https://github.com/user-attachments/assets/0804386e-b5d3-4015-a577-86f3df1046f4)
Many thanks for your consideration!

**Reply by SeppeDeWinter** [reply]
Hi @Melody-cell 

You can find it on GEO: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE210745

Best,

Seppe

**Reply by Melody-cell** [reply]
OK, I have found it now, thanks a lot!
